IFC4x3/Properties/b/BoreholeElevation_0zNwWthFTDjuOnS8GoqDch/Documentation.md

```diff
@@ -0,0 +1 @@
+If present, the property reports the elevation data, in metres, for the borehole (i.e., wellbore) start point. This is a compromise approach to allow for delivery of legacy 2D data without elevation data, and for software that cannot process a 3D GM_Point.
```

Comment:
```suggestion
If present, the property reports the elevation data in meters for the borehole (i.e., wellbore) start point. This is a compromise approach to allow for delivery of legacy 2D data without elevation data, and for software that cannot process a 3D GM_Point.
```

Reply:
IFC file do not come in 2D geometry only.

Reply:
The change proposed ('meters') is fine